aspnetcore/security/authorization/razor-pages-authorization.md

```diff
@@ -16,7 +16,7 @@ One way to control access in your Razor Pages app is to use authorization conven
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/security/authorization/razor-pages-authorization/samples) ([how to download](xref:index#how-to-download-a-sample))
 
-The sample app uses [cookie authentication without ASP.NET Core Identity](xref:security/authentication/cookie). The concepts and examples shown in this topic apply equally to apps that use ASP.NET Core Identity. To use ASP.NET Core Identity, follow the guidance in <xref:security/authentication/identity>.
+The sample app uses [cookie authentication without ASP.NET Core Identity](xref:security/authentication/cookie). The concepts and examples shown in this topic apply equally to apps that use ASP.NET Core Identity. To use ASP.NET Core Identity, follow the guidance in <xref:security/authentication/identity>. If pages normally configured for anonymous access should be restricted, see the [Authorize access to anonymous Identity pages](#authorize-access-to-anonymous-identity-pages) section.
```

Comment:
~ If pages normally configured for anonymous access should be restricted, see the [Authorize access to anonymous Identity pages](#authorize-access-to-anonymous-identity-pages) section.~

That's confusing and not helpful. Delete.